### ISSUE TITLE
Calculate the spacing between two polygons with arcs

### ISSUE THREAD

**Issue by Joker0503** [KLayout/klayout#1689]
Hello, can I calculate the spacing between two polygons with arcs when using the drc function？

**Comment by klayoutmatthias** [comment]
Please provide more details, like a screen shot, some test case or a more detailed explanation.

**Comment by Spraveen8-chary** [comment]
Yes, you can use geometric libraries like Shapely in Python to perform such calculations. Define the polygons, then use functions to compute distances and check if they meet your design rules.
